package.json

```diff
@@ -48,6 +47,7 @@
     "@types/node-fetch": "^2.6.6",
     "auto": "^10.3.0",
     "babel-jest": "^27.0.6",
+    "chromatic": "^10.5.1",
```

Comment:
Is there a reason we now depend upon `chromatic`?

Reply:
That's the recommendation of our new setup. But now that you mention it, we're using the GitHub action, which doesn't care about this at all. I can remove it.